src/main/java/com/adyen/Client.java

```diff
@@ -67,6 +67,9 @@ public class Client {
     public static final String DATA_PROTECTION_ENDPOINT_LIVE = "https://ca-live.adyen.com/ca/services/DataProtectionService";
     public static final String POS_TERMINAL_MANAGEMENT_VERSION = "v1";
     public static final String DATA_PROTECTION_VERSION = "v1";
+    public static final String BALANCE_PLATFORM_ENDPOINT_TEST = "https://balanceplatform-api-test.adyen.com/bcl";
```

Comment:
This comes in conflict with [transfers](https://docs.adyen.com/api-explorer/transfers/3/post/transfers) as it is already part of the `https://balanceplatform-api-test.adyen.com/btl/`. Could we make it generic?